samples/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+plugins {
+    id("convention.detekt")
+}
+
+buildscript {
```

Comment:
Too many workarounds )))

Reply:
It's till 7.2, all fixed according to github milestone